geziyor.go

```diff
@@ -349,3 +392,29 @@ func (g *Geziyor) startExporters() {
 		}()
 	}
 }
+
+func (g *Geziyor) startWorkers() {
+	if g.Opt.ConcurrentRequests != 0 {
```

Comment:
If `ConcurrentRequests` would be 1 by default, you could removed the entire alternative logic section